mlkem/indcpa.c

```diff
@@ -155,34 +157,69 @@ static unsigned int rej_uniform(int16_t *r,
 #define GEN_MATRIX_NBLOCKS ((12*KYBER_N/8*(1 << 12)/KYBER_Q + XOF_BLOCKBYTES)/XOF_BLOCKBYTES)
 // Not static for benchmarking
 void gen_matrix(polyvec *a, const uint8_t seed[KYBER_SYMBYTES], int transposed) {
-    unsigned int ctr, i, j, k;
-    unsigned int buflen, off;
-    uint8_t buf[GEN_MATRIX_NBLOCKS * XOF_BLOCKBYTES + 2];
+    unsigned int ctr, i, j;
+    unsigned int buflen;
+    uint8_t bufx[KECCAK_WAY][GEN_MATRIX_NBLOCKS *XOF_BLOCKBYTES + 2];
```

Comment:
I have no idea why the buffer need to `+2`. Does anyone know why?

Reply:
Track link: https://github.com/pq-crystals/kyber/issues/79

Reply:
Indeed seems to be an artifact from the Kyber90s version. Please delete.
Please also fix the spaces around `*` - it triggers my OCD.

Reply:
Yeah, my homebrew astyle reformat the `*`. I have to manually add the space. Update soon.

Reply:
updated.